static/src/stylesheets/components/pasteup-forms/src/_forms.scss

```diff
@@ -60,246 +96,276 @@ category: Common
  *
  * @group forms
  */
-.form {
-    margin-top: $gs-baseline*2;
-    margin-bottom: $gs-baseline*2;
-}
 
-.form__heading {
-    @include fs-bodyHeading(3);
-    margin-left: 0;
-    margin-right: 0;
-}
+// Form
+// ============================================================================
 
-.form__note,
-.form-field__note {
-    font-family: $pasteup-forms-fields-font-family;
-    font-size: 14px;
-    margin-bottom: 8px;
+.form {
+    margin-bottom: 0;
+    border-top: $pasteup-forms-form-border-top;
 }
 
+// Form Fieldset
+// ============================================================================
+
 .fieldset {
     border: 0;
-    border-top: $pasteup-forms-fieldset-border-top;
-    display: table;
-    padding: (($gs-baseline/3)*4 - 1) 0 ($gs-baseline*2);
+    padding: 0;
     margin: 0;
+    border-bottom: $pasteup-forms-fieldset-border-bottom;
+    display: table;
```

Comment:
Yeah it turns out that people have been updating the components directly in the `frontend` repo rather than doing releases. So me updating them properly seems to have gotten a whole bunch of updates.